pom.xml

```diff
@@ -94,8 +94,8 @@
 		<!-- NB: Deploy releases to the SciJava Maven repository. -->
 		<releaseProfiles>deploy-to-scijava</releaseProfiles>
 
-		<flimlib.version>2.1.2-SNAPSHOT</flimlib.version>
-		<flimj-ops.version>2.1.2-SNAPSHOT</flimj-ops.version>
+		<flimlib.version>2.1.1</flimlib.version>
+		<flimj-ops.version>2.1.1</flimj-ops.version>
```

Comment:
Ahh, I didn't notice this before merging. For future reference, the main branch should never have any commits with SNAPSHOT dependencies on it, even in the history. Otherwise, if you check out those revisions and try to build them again later, they will fail to build because those snapshots will no longer exist, or have different code backing them (since snapshots are mutable).

Suppose you need a new flimlib 2.2.0 with some new functionality, but that project is still on 2.2.0-SNAPSHOT with the 2.2.0 final release not yet cut. It's OK to use 2.2.0-SNAPSHOT snapshot on a topic branch here before flimlib 2.2.0 is finalized. You can go back and forth, changing flimlib and flimj-ui together. But then once you have solidified how flimlib is going to work, flimlib 2.2.0 should get released, and the history here should be rewritten (e.g. via `git rebase -i`) to insert a commit updating `flimlib.version` to 2.2.0, and all subsequent commits relying on it should be tested to make sure they compile and run versus that new release. It's annoying, I know, but without it, there are irreproducible builds in the flimj-ui history, and tools like `git bisect` become less effective at finding bugs later.

Reply:
Noted, and sorry for any trouble caused...